### PR TITLE
Update naming to to Valkey

### PR DESCRIPTION
**Pull request by madolson**
Documentation references should use `Valkey` while server and cli references are all under `valkey`.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,13 +5,13 @@
...
-placeholderkv-cli
-placeholderkv-sentinel
-placeholderkv-server
+valkey-benchmark
+valkey-check-aof
+valkey-check-rdb
+valkey-check-dump
+valkey-cli
+valkey-sentinel
+valkey-server
```
Comment (zuiderkwast) [inline .gitignore]: Cover both sets of filenames/symlinks?

```suggestion
*-benchmark
*-check-aof
*-check-rdb
*-check-dump
*-cli
*-sentinel
*-server
```

File: .github/workflows/daily.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ on:
         default: ''
       use_repo:
         description: 'repo owner and name'
-        default: 'placeholderkv/placeholderkv'
+        default: 'valkey-io'
```
Comment (PingXie) [inline .github/workflows/daily.yml]: valkey-io/valkey

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -19,9 +19,9 @@ There is also an active community of Redis users at Stack Overflow:
 
     https://stackoverflow.com/questions/tagged/redis
 
-Issues and pull requests for documentation belong on the placeholderkv-doc repo:
+Issues and pull requests for documentation belong on the valkey-doc repo:
 
-    https://github.com/placeholderkv/placeholderkv-doc
+    https://github.com/valkey-io-doc
```
Comment (zuiderkwast) [inline CONTRIBUTING.md]:
```suggestion
    https://github.com/valkey-io/valkey-doc
```

File: README.md [hunk 4]
```diff
@@ -1,22 +1,23 @@
 This README is under construction as we work to build a new community driven high performance key-value store.
 
-This project was forked from the open source Redis project right before the transition to their new source available licenses.
+This project was forked from the open source Redis project right before the transition to their new source available licenses. For now we intend to keep the
```
Comment (zuiderkwast) [inline README.md]: Incomplete sentence...?

File: .github/workflows/daily.yml (hunk 2 above)
Comment (madolson) [reply .github/workflows/daily.yml]:
```suggestion
        default: 'valkey-io/valkey'
```

**Review by zuiderkwast — COMMENTED** [review]
Almost LGTM

File: .gitignore (hunk 1 above)
Comment (madolson) [reply .gitignore]: This is a good idea!

File: README.md [hunk 5]
```diff
@@ -65,7 +66,7 @@ command in order to really clean everything and rebuild from scratch:
 
     % make distclean
 
-This will clean: jemalloc, lua, hiplaceholderkv, linenoise and other dependencies.
+This will clean: jemalloc, lua, hiredis, linenoise and other dependencies.
```
Comment (hpatro) [inline README.md:69]: Do we need to fork hiredis repository at some point?

File: README.md (hunk 4 above)
Comment (madolson) [reply README.md]: Yeah, trying to bite off too many updates before the announcement.

File: CONTRIBUTING.md (hunk 3 above)
Comment (PingXie) [inline CONTRIBUTING.md]: "valkey-io/valkey-doc"

File: README.md [hunk 6]
```diff
@@ -118,64 +119,64 @@ To build with support for the processor's internal instruction clock, use:
...
-Running PlaceHolderKV with TLS:
+Running valkey with TLS:
 ------------------
 
 Please consult the [TLS.md](TLS.md) file for more information on
-how to use PlaceHolderKV with TLS.
+how to use valkey with TLS.
 
-Playing with PlaceHolderKV
+Playing with valkey
```
Comment (hpatro) [inline README.md]: General comment - Unsure the choice of upper case/lower case first letter in this PR. I see Valkey and valkey used interchangeably. However, I think for the binary references we should use valkey.

File: README.md (hunk 5 above)
Comment (madolson) [reply README.md:69]: I don't know, for API compatibility we can probably keep it as long as we want, but we'll want to do something.

File: README.md (hunk 6 above)
Comment (madolson) [reply README.md]: Ok, I tried to be more consistent. Binaries are all lower case and naming is capitalized.

File: README.md (hunk 5 above)
Comment (PingXie) [reply README.md:69]: +1 to keep the original name. It is functional

File: README.md (hunk 6 above)
Comment (PingXie) [reply README.md]: I think this needs to be "Valkey"

File: .github/workflows/coverity.yml [hunk 7]
```diff
@@ -8,24 +8,24 @@ on:
...
     steps:
     - uses: actions/checkout@main
     - name: Download and extract the Coverity Build Tool
       run: |
-          wget -q https://scan.coverity.com/download/cxx/linux64 --post-data "token=${{ secrets.COVERITY_SCAN_TOKEN }}&project=placeholderkv-unstable" -O cov-analysis-linux64.tar.gz
+          wget -q https://scan.coverity.com/download/cxx/linux64 --post-data "token=${{ secrets.COVERITY_SCAN_TOKEN }}&project=valkey-unstable" -O cov-analysis-linux64.tar.gz
           mkdir cov-analysis-linux64
           tar xzf cov-analysis-linux64.tar.gz --strip 1 -C cov-analysis-linux64
-    - name: Install placeholderkv dependencies
+    - name: Install valkey dependencies
```
Comment (PingXie) [inline .github/workflows/coverity.yml]: Valkey

**Comment by roshkhatri** [comment]
Places in 00-RELEASENOTES `redis`->`Valkey`

File: .gitignore [hunk 8]
```diff
@@ -5,13 +5,13 @@
...
 *.log
 dump.rdb
-placeholderkv-benchmark
-placeholderkv-check-aof
-placeholderkv-check-rdb
-placeholderkv-check-dump
-placeholderkv-cli
-placeholderkv-sentinel
-placeholderkv-server
+*-benchmark
```
Comment (PingXie) [inline .gitignore:8]: valkey?

File: README.md [hunk 9]
```diff
@@ -118,64 +119,64 @@ To build with support for the processor's internal instruction clock, use:
...
 If you want to see a more verbose output, use the following:
 
     % make V=1
 
-Running PlaceHolderKV
+Running Valkey
 -------------
 
-To run PlaceHolderKV with the default configuration, just type:
+To run valkey with the default configuration, just type:
```
Comment (PingXie) [inline README.md]: I think this should be "Valkey" or "valkey-server"?

File: .gitignore (hunk 8 above)
Comment (madolson) [reply .gitignore:8]: This is server agnostic no?

**Review by PingXie — APPROVED** [review]
Thanks Madelyn!

**Comment by LiiNen** [comment]
Congrats for official name Valkey!

It seems like there is somewhere missed, even if it is just a markdown file.

- SECURITY.md
```
# Security Policy

PlaceHolderKV is under development, more information will become available.

## Reporting a Vulnerability

If you believe you've discovered a serious vulnerability, please contact Madelyn
at placeholderkv@gmail.com
```

also, official mail address is left as "placeholderkv@gmail.com", not only above, also in file below:
- CODE_OF_CONDUCT.md

p.s left it as comment, not PR, cause this is a very small point, and hope this to be changed together with mail address :)

**Comment by madolson** [comment]
Thanks! I did leave that intentionally as we didn't have the new email setup. Once that has been setup I'll update this page.

**Pull request merged**